lib/utils.js

```diff
@@ -36,7 +36,9 @@ export function grammarToLanguage(grammar: ?atom$Grammar) {
     l => l.toLowerCase() === grammarLanguage
   );
 
-  return kernelLanguage ? kernelLanguage.toLowerCase() : grammarLanguage;
```

Comment:
I'm chasing another bug; even with this PR `{ "babel": "javascript"}` doesn't work.